wordpress-18-04/template.json

```diff
@@ -0,0 +1,88 @@
+
+{
+  "variables": {
+    "do_api_token": "{{env `DIGITALOCEAN_API_TOKEN`}}",
+    "image_name": "wordpress-18-04-snapshot-{{timestamp}}",
+    "apt_packages": "apache2 fail2ban libapache2-mod-php mysql-server php7.4 php7.4-apcu php7.4-bz2 php7.4-curl php7.4-gd php7.4-gmp php7.4-intl php7.4-json php7.4-mbstring php7.4-mysql php7.4-pspell php7.4-soap php7.4-tidy php7.4-xml php7.4-xmlrpc php7.4-xsl php7.4-zip postfix python-certbot-apache software-properties-common unzip",
+    "appiication_name": "WordPress",
+    "appiication_version": "5.5.1",
```

Comment:
We are going to explicitly select which version of WordPress we want to install. We should automate triggering a new build anytime a new WordPress version is released.